babel.config.js

```diff
@@ -16,6 +16,7 @@ module.exports = (api, targets) => {
       ],
     ],
     plugins: [
+      '@babel/plugin-transform-runtime',
```

Comment:
we don't need this?

Reply:
I might be wrong, but I think we do -> https://github.com/pmndrs/jotai/pull/378#discussion_r602957217

I set this as I wanted to get rid of the warning when bundling.